pkg/api/v1alpha1/vspheremachineconfig_webhook.go

```diff
@@ -96,6 +89,15 @@ func validateImmutableFieldsVSphereMachineConfig(new, old *VSphereMachineConfig)
 
 	vspheremachineconfiglog.Info("Machine config is associated with management cluster's control plane or etcd", "name", old.Name)
 
+	if old.IsControlPlane() || old.IsEtcd() {
```

Comment:
do we still need this? starting l90 is already "management cluster's control plane or etcd"

Reply:
fixed.